### PR TITLE
added remain algorithm for JWT

### PR DESCRIPTION
**Pull request by Code-Hex**
### Author should do the followings, if applicable

- [x] Add tests
- [x] Run tests
- [x] `yarn denoify` to generate files for Deno

### CODE REVIEW

**Comment by Code-Hex** [comment]
I will send one more PR after this PR has been merged!

File: deno_dist/utils/jwt/jwt.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ const decodeJwtPart = (part: string): unknown =>
   JSON.parse(utf8Decoder.decode(decodeBase64Url(part)))
 
 const param = (name: AlgorithmTypeName): AlgorithmParams => {
-  switch (name.toUpperCase()) {
```
Comment (Code-Hex) [inline deno_dist/utils/jwt/jwt.ts:23]: This is non-secure because no exact comparison is made 😇

Comment (yusukebe) [reply deno_dist/utils/jwt/jwt.ts:23]: You are right!

**Review by yusukebe — APPROVED** [review]
Looks good to me!

**Comment by yusukebe** [comment]
@Code-Hex Thanks for the PR!

@metrue Could you review this one too?

**Comment by metrue** [comment]
LGTM

**Review by metrue — APPROVED** [review]

**Comment by yusukebe** [comment]
Thanks! Merge now.

**Pull request merged**